From 510e2543dffebf0e5c1203ff6b1ba6c488d6b48d Mon Sep 17 00:00:00 2001
From: jsecchiero <secchierojacopo@gmail.com>
Date: Thu, 16 May 2019 21:42:46 +0200
Subject: [PATCH] Add ignore_assert_errors to "kube-master, ...

... kube-node or etcd is empty" task
As a assert must be ignored if ignore_assert_errors is true
---
 roles/kubernetes/preinstall/tasks/0020-verify-settings.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/roles/kubernetes/preinstall/tasks/0020-verify-settings.yml b/roles/kubernetes/preinstall/tasks/0020-verify-settings.yml
index e7184d4a..4522a82e 100644
--- a/roles/kubernetes/preinstall/tasks/0020-verify-settings.yml
+++ b/roles/kubernetes/preinstall/tasks/0020-verify-settings.yml
@@ -7,6 +7,7 @@
     - kube-node
     - etcd
   run_once: true
+  ignore_errors: "{{ ignore_assert_errors }}"

 - name: Stop if non systemd OS type
   assert:
--
2.20.1
